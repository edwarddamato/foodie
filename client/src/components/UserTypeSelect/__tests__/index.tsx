import * as React from 'react';
import { UserTypeSelect } from '../index';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const user = {
    Name: 'Edward D\'Amato',
    Location: 'London',
    Bio: 'Moo'
  };
  const userTypeSelect = create(
    <UserTypeSelect user={user} />
  ).toJSON();
  expect(userTypeSelect).toMatchSnapshot();
});