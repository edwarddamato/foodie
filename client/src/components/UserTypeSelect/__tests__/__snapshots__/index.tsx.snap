// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="component_user-select"
>
  <div
    className="component_user-select-title"
  >
    What are you?
  </div>
  <ul
    className="component_user-select-types"
  >
    <li
      className="component_user-select-item"
    >
      <button
        className="component_user-select-button"
      >
        I'm a chef!
      </button>
    </li>
    <li
      className="component_user-select-item"
    >
      <button
        className="component_user-select-button"
      >
        I'm hungry!
      </button>
    </li>
  </ul>
</section>
`;
